docs/volunteering/writing/editing-bobadocs.md

```diff
@@ -0,0 +1,158 @@
+---
+sidebar_position: 1
+---
+
+# Editing BobaDocs
+
+Did you know that you can edit BobaDocs - the website that you're looking at *right now* - without ever having to touch a command line? This guide will walk you through it.
+
+**You will need a GitHub account to do this.**
+
+## Step 0: Setting up your git fork & branch
+
+### Forking
+
+**If you already have a fork of BobaDocs, replace this section with "Keeping your repo updated" in the [Appendix](./editing-bobadocs-appendix).**
+
+The files that this website runs off of (known collectively as the site's `repo`) are only allowed to be edited by the BobaLord: this prevents random people from showing up and screwing up the site. This also means that in order to make any changes, you'll need your own copy of the files, known as a `fork`.
+
+First, **make sure you're logged in to GitHub.** Once you're logged in, navigate to the main repo for the site at [https://github.com/essential-randomness/bobadocs](https://github.com/essential-randomness/bobadocs). You can also get there by clicking the "GitHub" link on the top right corner of the site.
+
+Once there, on the top right corner of the GitHub page, you should see a `Fork` button. Click it. You will be automatically redirected to your fork.
+
+![a cropped screenshot of the bobadocs github page, with a red arrow pointing to the fork button](/img/volunteer/bobadocsfork.jpg)
+
+### Branching
+
+In git, a `branch` is a separate version of a repo that can be edited without affecting the main repo. It lets developers separate what they're working on, so problems that arise when working on one issue don't affect any other work that may be being done concurrently. Making good use of branches is key to keeping a project history easy to understand. It also keeps multiple people working on the same project from stepping on each other's toes.
+
+To create a new branch, click the dropdown labelled "main" on the left side of the screen, enter a short, descriptive name for your repo in the textbox, and click "Create branch".
+
+![a cropped screenshot of the bobadocs github page, with red arrows pointing to relevant spots on the branch dropdown](/img/volunteer/bobadocsbranch.jpg)
+
+## Step 1: Navigating to GitHub's editor
```

Comment:
When it comes to steps, I would use an active voice: s/"Navigating"/"Navigate".

With that said, maybe it would be more like "navigate to the file to edit"? "find the page to edit"? I don't like either, but hopefully you get me.

Also, when a section has alternative methods, we should help people decide which method they prefer upfront (though in this case they're fairly equivalent). Personal preference, maybe, but I would also have the least complicated one as the first (to me this would be from the actual documentation website in the case of writers). 

 So in this case:

> ## Step 1: Find the right file
> 
> There are two ways to find the file corresponding to the page you want to edit: from the bobadocs website, or from within the github website. [[I really don't know what to say as far as the difference goes. For me they're somewhat equivalent, which can also be what we say.]]
> 
> ### Method 1 - Find file from BobaDocs Website
> 
> ### Method 2 - Find file from your fork